common.gypi

```diff
@@ -39,7 +39,7 @@
 
     # Reset this number to 0 on major V8 upgrades.
     # Increment by one for each non-official patch applied to deps/v8.
-    'v8_embedder_string': '-node.6',
+    'v8_embedder_string': '-node.7',
```

Comment:
There's a typo in the commit log (s/in all platforms/on all platforms/) and I don't see that `-Wbraced-scalar-init` it mentions anywhere.

Reply:
Thanks, I'll fix that